.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+Thanks for helping out! :smile:
+
+##Issue:
+1. Make sure the issue hasn't been raised yet.
+2. Include Screnshots or GIFs whenever possible.
+3. Don't be rude, describe your problem cool.
+
+
+##Pull Request:
+1. Use the present tense.
+2. References issues whenever possible.
+3. Make your code clean.(jsDoc would be great!)
```

Comment:
Here I would remove the JSDoc reference as it will maybe inforce people use JSDoc while we're not using it at the moment. I would refactor it like this: Try to make your code clean and follow the eslint rules observed by the project. Is it ok with you? That would be my last comment, I think :grin: